treeherder/webapp/api/performance_serializers.py

```diff
@@ -397,6 +397,21 @@ def get_name(self, value):
         return '{} {} {}'.format(test_suite, value['option_name'], value['extra_options'])
 
 
+class PerformanceAlertSummaryTasksSerializer(serializers.Serializer):
+    id = serializers.IntegerField(source='alert_summary_id')
+    tasks = serializers.ListField(child=serializers.CharField(), default=[])
+
+    def validate(self, data):
+        try:
+            PerformanceAlertSummary.objects.get(id=data['alert_summary_id'])
+        except PerformanceAlertSummary.DoesNotExist:
+            raise serializers.ValidationError(
+                {'message': 'PerformanceAlertSummary does not exist.'}
+            )
+
+        return data
```

Comment:
Based on the previous comment, I believe that the serializer declared in the `serializer_class` field should look something like:
```suggestion
class PerformanceAlertSummaryTasksSerializer(serializers.Serializer):
    id = serializers.IntegerField(source='alert_summary_id')
    tasks = serializers.ListField(child=serializers.CharField(), default=[])

      class Meta:
        model = PerformanceDatum
        fields = ['id', 'tasks']
```

Reply:
This method(using the class Meta, model field and fields) should use the `serializers.ModelSerializer` method.
I didn't use the `serializers.ModelSerializer` because many join tables existed.
If I use the `ModelSerializer`, I should make some unnecessary nested serializers such as job and job_type.
I think, In this case, using `ModelSerializer` might make the serializer more complicated, Which way would be better?

Reply:
I haven't actually tried to test the implementation, but I don't think you need to add nested serializers such as job and job_type.